CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [master](https://github.com/dallinger/dallinger/tree/master) (xxxx-xx-xx)
+- Changed: rename experiment method `on_assignment_submitted_to_recruiter` to
```

Comment:
Wondering if this will fit the bill... I added a check to `dallinger verify`:

```
$ dallinger verify

❯❯ Verifying current directory as a Dallinger experiment...
✓ config.txt is PRESENT
✓ experiment.py is PRESENT
✓ Size OK at 0MB (max is 50MB)
✓ experiment.py defines 1 experiment
✗ experiment.py overrides a method that has been renamed.
        Old name: on_assignment_submitted_to_recruiter
        New name: on_recruiter_submission_complete
Please rename your method accordingly.
✓ no file conflicts

❯❯ ☹ Some problems were found.
```

This will also block running the experiment:
```
$ dallinger debug --verbose
✓ config.txt is PRESENT
✓ experiment.py is PRESENT
✓ Size OK at 0MB (max is 50MB)
✓ experiment.py defines 1 experiment
✗ experiment.py overrides a method that has been renamed.
        Old name: on_assignment_submitted_to_recruiter
        New name: on_recruiter_submission_complete
Please rename your method accordingly.
✓ no file conflicts
Usage: dallinger debug [OPTIONS]
Try 'dallinger debug -h' for help.

Error: The current directory is not a valid Dallinger experiment.
```

Reply:
Excellent!